stable/artifactory/templates/artifactory-statefulset.yaml

```diff
@@ -530,6 +530,23 @@ spec:
 {{ toYaml .Values.filebeat.resources | indent 10 }}
         terminationGracePeriodSeconds: {{ .Values.terminationGracePeriod }}
       {{- end }}
+      {{- if .Values.fluentd.enabled }}
+      - name: {{ .Values.fluentd.name }}
+        image: "{{ .Values.fluentd.image.repository }}:{{ .Values.fluentd.image.version }}"
+        imagePullPolicy: {{ .Values.fluentd.image.pullPolicy }}
+        volumeMounts:
+        - name: fluent-conf
+          mountPath: /fluent-bit/etc/
+        - name: artifactory-volume
+          mountPath: "{{ .Values.artifactory.persistence.mountPath }}"
+        livenessProbe:
+{{ toYaml .Values.fluentd.livenessProbe | indent 10 }}
```

Comment:
Since you put this option, add it also to the template in `values.yaml`.  Can be commented out, but so users will see it and opt in if they want to.

Reply:
I looked into this the stable/fluent-bit chart in general does not have the readiness or liveness probe but I have seen some people online add a http input to their fluent-bit conf then they can curl this endpoint as an alternative liveness / readiness probe to fluent bit. Ill disable for now and leave a comment letting people know they can add it this way if desired.